Documentation/thanos.md

```diff
@@ -86,7 +86,7 @@ responsible for compactions on a global, object storage level.
 ## Thanos Ruler
```

Comment:
Hey @heliapb thanks for taking care of this.

Just to fix the current CI issues, could you run the following command?

```bash
make --always-make format generate
```

This will update the and format auto-generated docs

Reply:
done, pls review